client/dashboard/data/constants.ts

```diff
@@ -8,6 +8,7 @@ export enum DotcomFeatures {
 	SFTP = 'sftp',
 	SSH = 'ssh',
 	SITE_PREVIEW_LINKS = 'site-preview-links',
+	SITE_STAGING_SITES = 'staging-sites',
```

Comment:
Nit: why not just `STAGING_SITES` for consistency


```suggestion
	STAGING_SITES = 'staging-sites',
```

Reply:
because the previous one is `SITE_PREVIEW_LINKS` and we also use `FEATURE_SITE_STAGING_SITES` in other context 😂

Reply:
The strings come from backend, so it's better to keep them as-is 😄